tests/test__utils.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+from __future__ import absolute_import
+
+import pytest
+
+import numpy as np
+
+import dask_distance._utils
+
+
+def test_import_toplevel():
```

Comment:
This should have been named better, but was not. Fixing in PR ( https://github.com/jakirkham/dask-distance/pull/22 ).